File: Linguagem C++/Estrutura Condicional/Aumento.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    double salarioAtual,salarioNovo,aumento;
    int porcentagem;

    cout << "Digite o salario da pessoa: ";
    cin >> salarioAtual;

    if (salarioAtual <=1000){
        porcentagem = 20;
    }else if(salarioAtual <= 3000){
        porcentagem = 15;
    }else if(salarioAtual <= 8000){
        porcentagem = 10;
    }else {
        porcentagem = 5;
    };

    aumento = (salarioAtual / 100) * porcentagem;
    salarioNovo = salarioAtual + aumento;

    cout << fixed << setprecision(2);
    cout << "Novo salario = " << salarioNovo << endl;
    cout << "Aumento = " << aumento << endl;
    cout << "Porcentagem = " << porcentagem << "%" << endl;

    return 0;
}

File: Linguagem C#/Condicional/Aumento.cs
using System;
using System.Globalization;

namespace Aumento {
    class Program {
        static void Main(string[] args) {
            CultureInfo CI = CultureInfo.InvariantCulture;

            double salario, novoSalario, aumento;
            int porcentagem;

            Console.Write("Digite o salario da pessoa: ");
            salario = double.Parse(Console.ReadLine(), CI);

            if (salario <= 1000.00) {
                porcentagem = 20;
            }else if (salario <= 3000.00) {
                porcentagem = 15;
            }else if (salario <= 8000.00) {
                porcentagem = 10;
            }else {
                porcentagem = 5;
            };

            aumento = (salario / 100) * porcentagem;
            novoSalario = salario + aumento;

            Console.WriteLine("Novo salario = "+novoSalario.ToString("F2",CI));
            Console.WriteLine("Aumeto = "+aumento.ToString("F2",CI));
            Console.WriteLine("Porcentagem = "+porcentagem+"%");
        }
    }
}
